core/src/thread_parker/linux.rs

```diff
@@ -27,6 +34,8 @@ type tv_nsec_t = libc::c_long;
 
 // Helper type for putting a thread to sleep until some other thread wakes it up
 pub struct ThreadParker {
+    // REVIEW: is it really worth it to gate this entire module on the existence
+    // of `AtomicI32` rather than using `AtomicUsize`?
```

Comment:
Using `AtomicUsize` would be incorrect on 64-bit platforms.

Reply:
Would it really? Sure it's not 32 bit it's actually 64, but why does that matter?

Reply:
(my point is that like elsewhere you could use `AtomicI32` everywhere here and on stable and/or older versions just have a typedef of `type AtomicI32 = AtomicUsize` or something like that)